tests/e2e/util.go

```diff
@@ -3976,13 +4055,24 @@ func sshExec(sshClientConfig *ssh.ClientConfig, host string, cmd string) (fssh.R
 func createPod(client clientset.Interface, namespace string, nodeSelector map[string]string,
 	pvclaims []*v1.PersistentVolumeClaim, isPrivileged bool, command string) (*v1.Pod, error) {
 	pod := fpod.MakePod(namespace, nodeSelector, pvclaims, isPrivileged, command)
-	pod.Spec.Containers[0].Image = busyBoxImageOnGcr
+	commands := []string{"Powershell.exe", "-Command", command}
+	if windowsEnv {
+		pod.Spec.Containers[0].Image = windowsLTSC2019Image
+		pod.Spec.Containers[0].Command = commands
+		pod.Spec.Containers[0].VolumeMounts[0].MountPath = pod.Spec.Containers[0].VolumeMounts[0].MountPath + "/"
+	} else {
+		pod.Spec.Containers[0].Image = busyBoxImageOnGcr
+	}
 	pod, err := client.CoreV1().Pods(namespace).Create(context.TODO(), pod, metav1.CreateOptions{})
 	if err != nil {
 		return nil, fmt.Errorf("pod Create API error: %v", err)
 	}
 	// Waiting for pod to be running.
-	err = fpod.WaitForPodNameRunningInNamespace(client, pod.Name, namespace)
+	if windowsEnv {
+		err = fpod.WaitForPodRunningInNamespaceSlow(client, pod.Name, namespace)
```

Comment:
WaitForPodRunningInNamespaceSlow and WaitForPodNameRunningInNamespace uses same set of parameter, If  the difference is only on the wait time , We can handle this in WaitForPodNameRunningInNamespace Method only  instead of writing a new method

Reply:
this is not a new method. It's existing one. Just used it here.

Reply:
removed WaitForPodNameRunningInNamespace, as pulling images on windows worker node. So long wait time removed.